src/potions.ts

```diff
@@ -460,18 +484,64 @@ const rufusPotion = new Potion($item`closed-circuit pay phone`, {
   },
 });
 
-export const wishPotions = Effect.all()
-  .filter((effect) => !effect.attributes.includes("nohookah"))
+export const wishPotions = validWishes.map(
+  (effect) =>
+    new Potion($item`pocket wish`, {
+      effect,
+      canDouble: false,
+      duration: 20,
+      use: (quantity: number) =>
+        new Array(quantity).fill(0).every(() => {
+          const s = getValidWishString(effect);
+          if (s === "") return;
```

Comment:
```suggestion
          if (!s) return;
```

Reply:
honestly, given that we always check against "is an empty string", using `null` or `undefined` instead of `""` is going to be easier to follow imo